tests/backend/test_blueprints.py

```
@@ -1,16 +1,29 @@
-from concurrent.futures import ThreadPoolExecutor
+from concurrent.futures import Future, ThreadPoolExecutor, wait
 from contextlib import contextmanager
+from threading import Event
 from typing import Dict, Optional, Type
```

Flake8 (7.2.0)

[error] 'typing.Type' imported but unused

(F401)